### DISCUSSION TITLE
Remove tracking pixels

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1537] [Feature Requests]
Hello!
Implement, please, functionality similar to what is now working on the famous service from DuckDuckGo.

Here is an article to better understand what I mean:
[ARTICLE](https://www.spreadprivacy.com/introducing-email-protection-beta/) 

You can add something like this bar on every email with had been modified (tracking pixels removed):
[SCREENSHOT](https://user-images.githubusercontent.com/122754090/212569610-55ac65d2-be95-4eb7-87fd-cd83e281faf1.jpeg)

**Comment by 4b15ed24f576** [comment]
Hi there, thank you for taking the time to provide us with your feedback regarding our product. Removing email tracking is indeed a very interesting feature and match our goal to contribute to a more secure and private Internet.

We will review your feedback and discuss it internally to determine if we can potentially implement it in our roadmap.

I will also leave this feature request open for more users to debate this request.

Thank you again for your valuable contribution.
